src/main/resources/application.yml

```diff
@@ -21,17 +21,6 @@ spring:
       resourceserver:
         jwt:
           jwk-set-uri: ${hmpps.auth.url}/.well-known/jwks.json
-      client:
-        registration:
-          prison-api:
-            provider: hmpps-auth
-            client-id: ${system.client.id}
-            client-secret: ${system.client.secret}
-            authorization-grant-type: client_credentials
-            scope: read,write
-        provider:
-          hmpps-auth:
-            token-uri: ${hmpps.auth.url}/oauth/token
 
```

Comment:
This last line above - is used to get the public key of the auth provider (hmpps-auth) so the validity of received tokens can be checked.

Reply:
Added the provider section back in